extras/createdevproject.py

```diff
@@ -0,0 +1,47 @@
+"""
+Modifies the project template to fit with local development
+"""
+import os
+import sys
+from misago.core import setup
+
+
+PROJECT_NAME = 'forum'
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+def main():
+    global PROJECT_NAME, SETTINGS_FILE
```

Comment:
Why is global used here? Especially `PROJECT_NAME`  should be in scope.

Reply:
Will remove. It's remains of some early sins when testing!